include/m/m_mtx.h

```diff
@@ -19,27 +19,6 @@ class mMtx_c {
     /* 802f1660 */ mMtx_c(f32 xx, f32 xy, f32 xz, f32 xw, f32 yx, f32 yy, f32 yz, f32 yw, f32 zx, f32 zy, f32 zz,
             f32 zw);
 
-    // not sure if this breaks anything but we need a matrix type
-    // with an inline copy assignment operator
-    mMtx_c &operator=(const mMtx_c &r) {
```

Comment:
This was a hack to get some m3d shadow code matching but it's entirely unclear if these are even the used data types...

Reply:
... so I turned this into an explicit `set` function.